realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
Another concern is this is not consistent with `Realm.getInstance()` which is based on a counter. (But actually I think `Realm.getInstance()` is the one should be changed ...) maybe we should just doc it here.